src/renderer/util/fileSystem.js

```diff
@@ -2,30 +2,29 @@ import path from 'path'
 import crypto from 'crypto'
 import { clipboard } from 'electron'
 import fs from 'fs-extra'
+import { statSync, constants } from 'fs'
+import cp from 'child_process'
+import { tmpdir } from 'os'
 import dayjs from 'dayjs'
 import { Octokit } from '@octokit/rest'
-import { ensureDirSync } from 'common/filesystem'
 import { isImageFile } from 'common/filesystem/paths'
-import cp from 'child_process'
-import { tmpdir } from 'os'
-import { unlink, writeFileSync, statSync, constants } from 'fs'
 import { isWindows, dataURItoBlob } from './index'
 import axios from '../axios'
 
-export const create = (pathname, type) => {
-  if (type === 'directory') {
-    return fs.ensureDir(pathname)
-  } else {
-    return fs.outputFile(pathname, '')
-  }
+export const create = async (pathname, type) => {
```

Comment:
There is no need to use `async` function if `await` sentence is at the last sentence, and fs.ensureDir already return a promise?

Reply:
Ok, makes sense. Should we keep `async` because JS has no type information and you need to know that the called functions are asynchronous?

Reply:
ok